tests/test.py

```diff
@@ -924,3 +931,89 @@ def test_01_cli_version(self):
         self.assertIn("libqpdf", stdout)
         self.assertIn("OS-", stdout)
         self.assertIn("OS_Version-", stdout)
+
+
+class TestBatch10(PdfArrangerTest):
+    """Test show-save-warnings config settings"""
```

Comment:
You are not testing show-save-warnings here, are you?

Reply:
Yes, thanks. This used to be `"""Test saving config settings"""` but I thought this is ambiguous (test the config settings for saving vs. test that config settings are correctly saved), so I decided to re-word it but got myself confused.  This should read 
```
class TestBatch10(PdfArrangerTest):
    """Test start-with-empty config settings""" 
```
which is easier to relate to the actual code. Will fix it.